documents/src/pages/elements/list.md

```diff
@@ -68,7 +68,7 @@ if (list) {
 
 ## Data property interface
 
-The `data` property of the `ef-list` use the [ListData](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/list/helpers/types.ts) interface for its data items.
+The `data` property of the `ef-list` use the [ListData](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/list/helpers/types.ts) interface that extend from [Data Item](./custom-components/utils/data-management#data-item) interface.
```

Comment:
This statement is wrong.

Reply:
Could you tell me why? I see ListData uses ItemData by default. ItemData extends DataItem.